src/components/Datatable/ElementCounter/ElementCounter.types.ts

```diff
@@ -0,0 +1,5 @@
+export interface ElementCounterProps {
+  totalLength: number;
```

Comment:
what about the case when the numbers are large? in the design for example the total is rendered as "1.05M" .

Reply:
ok @ajkl2533 I see you are using `formatNumber` and it works but renders 1.05m with small "m"

Reply:
maybe adding of this use case to storybook would be helpful

Reply:
sure guys :) fixed the uppercase and added abbreviated counts into SB